test_cases/mtg_with_detailed_prompts/2024-04.csv

```diff
@@ -0,0 +1,477 @@
+uuid,date,messages
```

Comment:
Situation:
We're a team of 6 people with the following userIds: 1,2,3,. to develop an application about a task management
Uuid1 is a dev lead:
in first week, show anxiety, feel so bad for the future development
in second week, show still anxiety
in third week, show find a little bit hope.
in forth week, because we have completed their tasks so show happy feelings.

Uuid2 is a designer:
in first week, show optimistic feelings and say kind words to other members.
in second week, show still less optimistic feelings
in third week, find dificulties in his tasks and show angry
in fourth week, show dissatisfaction because he cannot do in a way he wants

Uuid3 is a junior developer, consistent over weeks in motivation and normal feelings
over weeks, he shows consistent feelings

Conditions
We're having daily conversations on a chatting platform with various members on one day.
Please generate 5-10 conversational messages for each person for each day.
Please do this for 28 days.
Please deliver the data in a CSV format as a file for first 5 days
In each day, there should be 5~8 conversations.

Column1 should be uuid (int)
Column2 should be date (yyyy-mm-dd)
Column3 should be messages (str)

Sort the lines by dates and the conversation in order naturally.

Make each message to be between 50 and 200 characters and only write complete sentences with opinions and statements.

Don't sort them based on uuid.
Rather focus on the natural conversation order